packages/design-system/src/components/Drawer/DrawerManager.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { createContext, useState, useContext, useRef } from 'react';
+import uniqueId from 'lodash/uniqueId';
+
+export const DrawerContext = createContext(null);
+
+export const DrawerManager = (props: any) => {
```

Comment:
Oh, I just remembered that we were originally planning to call this `DrawerManagerProvider`. Was this an intentional change?

Reply:
Yeah I thought DrawerManager seemed a little easier to remember, and the Provider itself is abstracted into the component.. I'm happy to change it if you prefer but i do like it better being shorter.

Reply:
You brought up the other day that adding "Provider" made it clear what its role was and that it was following the Context Provider pattern. However, [React's own documentation](https://react.dev/learn/passing-data-deeply-with-context) doesn't seem to have an opinion on naming, so I'm fine with keeping it succinct 👍